extkeys/mnemonic.go

```diff
@@ -220,7 +240,7 @@ func (m *Mnemonic) ValidMnemonic(mnemonic string, language Language) bool {
 
 // WordList returns list of words for a given language
 func (m *Mnemonic) WordList(language Language) (*WordList, error) {
-	if m.wordLists[language] == nil {
+	if int(language) < 0 || int(language) > len(m.wordLists)-1 || m.wordLists[language] == nil {
```

Comment:
The previous code looked fine. I would probably write it like this:
```go
words, ok := m.wordLists[language]
if !ok {
  return nil, fmt.Errorf("language word list is missing (language id: %d)", language)
}

return words
```

but the previous version should work the same in this case.

Reply:
@adambabik I think `words, ok := m.wordLists[language]` would work with a map but in this case it's an array: `wordLists [totalAvailableLanguages]*WordList`, is that correct?

Reply:
Oh, shoot, I thought it's a map. Let's keep your code then! 

PS we can write `int(language) >= len(m.wordLists)`.